package.json

```diff
@@ -1,13 +1,19 @@
 {
   "name": "pbench_dashboard",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
I am not sure but the version in my deployed dashboard shows `2.1.1-0`

Reply:
How did you deploy it?

Reply:
@portante I deployed it on localhost, changing some parts of main.yml